src/tasks/docs/load-from-file.js

```diff
@@ -47,23 +47,23 @@ export default function loadSourcesFromLocal(model) {
         const fileName = path.basename(filePath); // имя файла (с расширением)
         const fileExt = path.extname(fileName); // расширение файла
 
-        const localFilePath = path.resolve(filePath);
+        const absFilePath = path.resolve(filePath);
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'filePath' is not defined.